### PR TITLE
Update auditing-solutions-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a new note stating that Microsoft Purview auditing is the new name for unified audit log.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9594

File: microsoft-365/compliance/auditing-solutions-overview.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ ms.custom: seo-marvel-apr2020
 
 Microsoft Purview auditing solutions provide an integrated solution to help organizations effectively respond to security events, forensic investigations, internal investigations, and compliance obligations. Thousands of user and admin operations performed in dozens of Microsoft 365 services and solutions are captured, recorded, and retained in your organization's unified audit log. Audit records for these events are searchable by security ops, IT admins, insider risk teams, and compliance and legal investigators in your organization. This capability provides visibility into the activities performed across your Microsoft 365 organization.
 
+> [!NOTE]
+> The Microsoft Purview auditing is the new name for unified audit log.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/auditing-solutions-overview.md]:
```suggestion
> Microsoft Purview auditing is the new name for the unified audit log.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Review by v-tophillips — APPROVED** [review]

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**